apps/documentation/src/routes/documentation/cli.mdx

```diff
@@ -21,6 +19,22 @@ Tuono is the CLI that provides all the needed commands to handle the full-stack
 
 To list all the available commands, run `tuono -h`.
 
+```bash
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>“React” is a proper noun and needs to be capitalized. (A_GOOGLE[1])
Suggestions: `React`
Rule: https://community.languagetool.org/rule/show/A_GOOGLE?lang=en-US&subId=1
Category: GRAMMAR
<!-- __reviewdog__:ChA2ZjIzYzgxNjJhYzFkODcyEgxMYW5ndWFnZVRvb2w= -->